x/stakeibc/keeper/unbonding_records.go

```diff
@@ -229,10 +295,16 @@ func (k Keeper) SweepAllUnbondedTokens(ctx sdk.Context) {
 			k.Logger(ctx).Info(fmt.Sprintf("processing epochUnbondingRecord %v", epochUnbondingRecord.EpochNumber))
 
 			// iterate through all host zone unbondings and process them if they're ready to be swept
+			// TODO Question, what should we do if one epochUnbondingRecord is erroneous?
```

Comment:
100% agree! I think the question here though is what should we do for 1 hostZone if, for whatever reason, one of the epochUnbondingRecords is an unexpected state. do we just ignore that one record and process the rest, or ignore the hostZone altogether?

Reply:
ohhh I had a feeling I was misinterpreting the question haha.

And just to confirm, you're referring to if the call `GetHostZoneUnbondingsByChainId` does not find any unbonding records?

Reply:
Yeah exactly! Although now that I'm thinking about it, I think this actually can happen (e.g. what if a hostZone was registered after another one). I'll change it so that it continues instead of exits

Reply:
Ah great catch!